src/status_im/ui/screens/add_new/open_dapp/styles.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.ui.screens.add-new.open-dapp.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
+  (:require [status-im.ui.components.colors :as colors]))
+
+(def main-container {:flex 1})
+
+(def enter-url
+  {:flex-direction    :row
+   :align-items       :center
+   :border-radius     8
+   :height            52
+   :background-color  colors/light-gray
+   :margin-horizontal 14
+   :margin-top        24})
+
+(defstyle url-input
+  {:flex               1
+   :font-size          15
+   :letter-spacing     -0.2
```

Comment:
We should have some constant for the `letter-spacing` (or at even better a component)

Reply:
yeah but not in this PR

Reply:
:+1:

Reply:
Can you create an issue to capture that?

Reply:
https://github.com/status-im/status-react/issues/3206